src/ArborX_BruteForce.hpp

```diff
@@ -91,14 +92,15 @@ class BasicBruteForce
   IndexableGetter _indexable_getter;
 };
 
-template <typename MemorySpace, typename BoundingVolume = Box>
-class BruteForce
-    : public BasicBruteForce<MemorySpace, Details::PairIndexVolume<Box>,
-                             Details::DefaultIndexableGetter, BoundingVolume>
+template <typename MemorySpace>
```

Comment:
This changes the number of template arguments and actually breaks backwards compatibility if someone was using `ArborX::BruteForce<MemorySpace, BoundingVolume>`. But to my knowledge, no one had done it, as we have not documented it.